docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,29 @@
+# AHA Prod Release Workflow
+Explaination of AHA Prod Release Workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+![AHA-PROD-JOB](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/aha_workflow.png?raw=true)
+
+## AHA-PROD-CI-CD
+
+This workflow allows you to manually trigger a GitHub Action with a input GitHub ``` Tag_name ```, without having to push or create a pull request
+This workflow uses two jobs: GitHub-ECR-Tag-Check and Deploy ECS to verify and deploy reancare-service release to aha-prod environment
+
+### JOBS
+
+#### GitHub-ECR-Tag-Check
```

Comment:
You are explaining two jobs here ECR Tag Check and Deploy ECS, but the example execution run screenshot you attached from GitHub shows only the "Deploy-ECS" job